pkg/restapi/operation/operations_test.go

```diff
@@ -630,14 +649,14 @@ func TestCreateDocument(t *testing.T) {
 		rr := httptest.NewRecorder()
 
 		urlVars := make(map[string]string)
-		urlVars[vaultIDPathVariable] = testVaultID
+		urlVars[vaultIDPathVariable] = testReferenceID
```

Comment:
Same comment as before - should really be some sort of testVaultID

Reply:
fixed